server/authz/rbac.go

```diff
@@ -20,49 +20,71 @@ import (
 	_ "embed"
 	"fmt"
 	"path"
+	"strings"
 
 	"github.com/casbin/casbin/v2"
 	"github.com/casbin/casbin/v2/model"
 	fileadapter "github.com/casbin/casbin/v2/persist/file-adapter"
 	"github.com/gin-gonic/gin"
+	"github.com/spf13/viper"
+
+	"github.com/numaproj/numaflow/pkg/shared/logging"
+	"github.com/numaproj/numaflow/server/authn"
 )
 
 var (
 	//go:embed rbac-model.conf
 	rbacModel string
+	logger    = logging.NewLogger().Named("server")
```

Comment:
why is the logger in the module scope? also, why not use https://github.com/numaproj/numaflow/blob/main/pkg/shared/logging/log.go#L28 ?